### PR TITLE
Adds in Fingersnapping

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Adds in a new emote: *snap

code largely from Urist: https://github.com/UristMcStation/UristMcStation/pull/153 but with fixes
Sound credit to @Nienhaus

- Makes you snap your fingers
 - Can't snap your fingers if both hands are gone/broken
 - 5% chance to fail and generate a comedic message+sound.
 - standard sound-related emote CD applies

### CODE REVIEW

**Comment by Iamgoofball** [comment]
:-1: holy hell that's way too many lines for a single emote that has no gameplay benefit

**Comment by SirBloodBuske** [comment]
Alright add in greaser jackets, and blue pants.. THEN WE CAN REENACT GREASE!

**Comment by FreeStylaLT** [comment]
oh no you di-in't :+1:

**Comment by Fox-McCloud** [comment]
@SirBloodBuske 

There's already leather jackets and jeans (of multiple types).

**Comment by SirBloodBuske** [comment]
@Fox-McCloud Alright good, all we need now is.. well.. a girl and a jukebox.

**Comment by Spacemanspark** [comment]
:+1:

**Comment by M3hillus** [comment]
Tunnel snakes rule!

**Comment by tigercat2000** [comment]
This is slightly amusing, but I don't really like the probability for the message

it feels like importing more goon memes to me

**Comment by SirBloodBuske** [comment]
@tigercat2000 we can add purring for tajarans next :3

**Comment by veganzombeh** [comment]
:+1:, even though the failure message is a bit... strange. It sounds like it would break their hand or something.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -720,6 +720,29 @@
...
+					right_hand_good = 1
+
+				if (!left_hand_good && !right_hand_good)
+					usr << "You need at least one hand in good working order to snap your fingers."
+					return
+
+				message = "<b>[src]</b> snaps \his fingers."
+				playsound(src.loc, 'sound/effects/fingersnap.ogg', 50, 1, -3)
+			else
+				message = "<span class='danger'><b>[src]</b> snaps \his fingers right off!</span>"
```
Comment (ZomgPonies) [inline code/modules/mob/living/carbon/human/emote.dm]: What does that mean? Does he take damage? Is he impeded in any way? Why would he be? Can you not snap your fingers 20 times? Unless there's some sort of gameplay impact that makes sense, I don't think this bit is necessary.

The rest of the emote is fine.

Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/emote.dm]: Doesn't take damage; just a silly little thing---and also discourage spamming it incessantly.

**Pull request merged**